### ISSUE TITLE
Handle 403 error

### ISSUE THREAD

**Issue by ksauzz** [materialsproject/api#817]
We sometimes observed https://api.materialsproject.org/heartbeat responds a 403 error, then mp-api just raises JSONDecodeError.
It would be nice to show users a 403 error, and what could happen such as rate limit or something.

```
JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

refs:
https://matsci.org/t/pymatgen-ext-matproj-mpresterror-rest-query-returned-with-error-status-code-403-content-berror-code-1020/47992
https://matsci.org/t/mpresterror-rest-query-returned-with-error-status-code-403/792

**Comment by munrojm** [comment]
Yes, I agree with this. The client should communicate better when the API service appears to be unreachable.

**Comment by esoteric-ephemera** [comment]
Should be fixed in [#1053](https://github.com/materialsproject/api/pull/1053): Tf the heartbeat page returns a 403, emit a warning linking to the status page, and the client should work as normal
